packages/grafana-ui/src/components/ConfirmModal/ConfirmModal.tsx

```diff
@@ -44,6 +44,8 @@ export interface ConfirmModalProps {
   onDismiss(): void;
   /** Alternative action callback */
   onAlternative?(): void;
+  /** Disable the confirm button if needed */
+  isConfirmButtonDissabled?: boolean;
```

Comment:
What if we just add a `disabled` prop that disables both the text entry and confirm button?

I _think_ we can keep the cancel button enabled? I think that's understable for consumers?